src/server/graphql/types/InvoiceChargeNextMonth.js

```diff
@@ -0,0 +1,28 @@
+import {GraphQLFloat, GraphQLInt, GraphQLNonNull, GraphQLObjectType} from 'graphql';
+import GraphQLISO8601Type from 'server/graphql/types/GraphQLISO8601Type';
+
+
+const InvoiceChargeNextMonth = new GraphQLObjectType({
```

Comment:
another great question! `InvoiceChargeNextMonth` is only a subset of an invoice. It only has 4 fields so when i'm in GraphiQL i shouldn't see eg `lines` as a valid option, since that won't exist!